Integrations/python/deephaven/TableManipulation/__init__.py

```diff
@@ -9,18 +9,18 @@
 import jpy
```

Comment:
I did a quick search of https://github.com/jjbrosnan/deephaven-core/blob/add-replayer-to-python/Integrations/python/deephaven/__init__.py for `TableManipulation`.  I see a number of places in that file that look like they might need changing as well.